### DISCUSSION TITLE
第6节课作业(3班)

### DISCUSSION THREAD

**Discussion by matrixgame2018** [InternLM/Tutorial#428] [作业]
第6节课作业(3班)

**Comment by SchweitzerGAO** [comment]
https://schweitzergao.github.io/2024/01/18/mmtutorial-6-assignment/

**Comment by xiexiaoshinick** [comment]
[[笔记和作业]6.OpenCompass 大模型评测 - 知乎 (zhihu.com)](https://zhuanlan.zhihu.com/p/679047918)

**Comment by hydexi0122** [comment]
https://zhuanlan.zhihu.com/p/679063645

**Comment by 443752009** [comment]
http://t.csdnimg.cn/0cin1

**Comment by Sampson425** [comment]
https://blog.csdn.net/m0_49908454/article/details/135748647

**Comment by GoldWaterFall** [comment]
https://www.yuque.com/woshijinshui/wwi5m1/ev4hlnwrgq9que1f?singleDoc# 《OpenCompass评测作业》

**Comment by Tallisgo** [comment]
http://t.csdnimg.cn/9ZLXY

**Comment by GoooHi** [comment]
https://zhuanlan.zhihu.com/p/679106108

**Comment by G-Mie** [comment]
https://blog.csdn.net/YYYYbhjbhkv_/article/details/135856872
